code/__HELPERS/icon_smoothing.dm

```diff
@@ -269,18 +269,11 @@ DEFINE_BITFIELD(smoothing_junction, list(
 	if(new_overlays.len)
 		A.add_overlay(new_overlays)
 
-
 ///Scans direction to find targets to smooth with.
 /atom/proc/find_type_in_direction(direction)
 	var/turf/target_turf = get_step(src, direction)
 	if(!target_turf)
 		return NULLTURF_BORDER
-
-	var/area/target_area = get_area(target_turf)
-	var/area/source_area = get_area(src)
-	if((source_area.area_limited_icon_smoothing && !istype(target_area, source_area.area_limited_icon_smoothing)) || (target_area.area_limited_icon_smoothing && !istype(source_area, target_area.area_limited_icon_smoothing)))
-		return NO_ADJ_FOUND
-
```

Comment:
Is this accidentally removed?

Reply:
Nope! We never go into this section of code